<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>508</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Venda e Controle de Ingressos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Inserir">
     <attribute name="title">
      <string extracomment="Inserir">Inserir</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_nameInput"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="IngressoBox">
        <item>
         <property name="text">
          <string>INTEIRA PISTA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MEIA PISTA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>INTEIRA FRONT</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MEIA FRONT</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_insert">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Inserir</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="tbl_data">
          <property name="minimumSize">
           <size>
            <width>487</width>
            <height>0</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Tipo do Ingresso</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>N° do Ingresso</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nome do Comprador</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="btn_NameSort">
          <property name="text">
           <string>Ordenar por Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_NumberSort">
          <property name="text">
           <string>Ordenar por Número do Ingresso</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>461</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="gb_all">
         <property name="title">
          <string>Porcentagem de Ingresso Vendidos no Total</string>
         </property>
         <widget class="QLabel" name="lbl_all">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>441</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_high">
         <property name="title">
          <string>Porcentagem da Categoria Mais Vendida</string>
         </property>
         <widget class="QLabel" name="lbl_high">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>441</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_low">
         <property name="title">
          <string>Porcentagem da Categoria Menos Vendida</string>
         </property>
         <widget class="QLabel" name="lbl_low">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>441</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="descri1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo de Ingresso Mais Vendido:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="descri2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo de Ingresso Menos Vendido:</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>206</x>
        <y>11</y>
        <width>280</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lbl_most_selled">
         <property name="text">
          <string>Não Foi Vendido Nenhum Ingresso Ainda, Sr(a) Usuário(a)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_few_selled">
         <property name="text">
          <string>Não Foi Vendido Nenhum Ingresso Ainda, Sr(a) Usuário(a)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuArquivo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
